rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/data/DataUtility.java

```diff
@@ -29,7 +29,9 @@
 
 import java.io.IOException;
 import java.util.ArrayList;
+import java.util.LinkedHashMap;
 import java.util.List;
+import java.util.Map;
```

Comment:
in this class, the toJobInfo(..) method should also be modified to include the submissionMode